### PR TITLE
Update WorldGuard.php

### PR DESCRIPTION
**Pull request by kenygamer**
#### **What does the PR do?**

I looked into your code and documented it using [PHPDoc](https://phpdoc.org) standards.

#### **Why should this PR be merged?**

This will make easier for developers to work around your code, learning from its structure.

#### **Are the changes in this PR tested**?

- [x] Yes
- [ ] No

#### **Extra Information**

[Following me](https://github.com/kenygamer) is much appreciated. While I'm not forcing anyone to do it, it gives me the incentive to continue my work on GitHub.

### CODE REVIEW

File: src/WorldGuard/WorldGuard.php [hunk 1]
```diff
@@ -155,12 +196,22 @@ public function sessionizePlayer(Player $player)
         $this->updateRegion($player);
     }
 
+    /**
+     * @param Position $pos
+     *
+     * @return string
+     */
     public function getRegionFromPosition(Position $pos)
```
Comment (Muqsit) [inline src/WorldGuard/WorldGuard.php]: This function can return `Region` too.

Comment (kenygamer) [reply src/WorldGuard/WorldGuard.php]: @Muqsit I get it. Please push the pull request to send another with the changes.

Comment (Muqsit) [reply src/WorldGuard/WorldGuard.php]: You can modify this one itself.

Comment (kenygamer) [reply src/WorldGuard/WorldGuard.php]: @Muqsit I deleted the forked repository.

**Pull request closed without merging**